src/audit-allowlist.json

```diff
@@ -21,8 +21,8 @@
             "reason": "indirect dependency of a few babel packages. should be looked at as a seperate ticket. see: https://trello.com/c/alXeUZpI/1918-fix-dependency-issues-with-babel-traverse"
         },
         {
-            "id": "CVE-2023-45857",
-            "reason": "axios is only used by interactives and will be removed soon. see: https://trello.com/c/zub5ECee/1916-remove-interactive-api-code-config-from-florence-and-dp-configs"
+            "id" : "CVE-2022-25883",
+            "reason": "Semver"
```

Comment:
Can you provide a bit more of an explanation of why this has been allowed?

Reply:
Wish i could remember why. It was added months ago https://github.com/ONSdigital/florence/pull/989/files

I'll have a look at the details and try add something more helpful